src/librustdoc/html/static/search.js

```diff
@@ -152,7 +152,7 @@ window.initSearch = function(rawSearchIndex) {
                 if (results[i].id > -1) {
                     var obj = searchIndex[results[i].id];
                     obj.lev = results[i].lev;
-                    if (isType !== true || obj.type) {
+                    if (!isType || obj.type) {
```

Comment:
The `isType` argument is never passed to the function, meaning it's completely useless. Can you remove it instead please?